python/cloudtik/core/_private/utils.py

```diff
@@ -2705,3 +2707,9 @@ def get_storage_config_for_update(provider_config):
     if "storage" not in provider_config:
         provider_config["storage"] = {}
     return provider_config["storage"]
+
+
+def get_pem_path_for_kubernetes(config):
```

Comment:
We should put this to Kubernetes provider. 
How this can avoided in common code using provider?  Is it possible to use provider bootstrap to set the private key file path just as AWS or GCP?